plugins/Dashboard/API.php

```diff
@@ -112,7 +113,21 @@ public function removeDashboard($idDashboard, $login='')
      */
     public function copyDashboardToUser($idDashboard, $copyToUser, $dashboardName = '')
     {
-        Piwik::checkUserHasSuperUserAccess();
+        Piwik::checkUserHasSomeAdminAccess();
+
+        // get users only returns users of sites the current user has at least admin access to
+        $users = Request::processRequest('UsersManager.getUsers');
+        $userFound = false;
+        foreach ($users as $user) {
+            if ($user['login'] === $copyToUser) {
+                $userFound = true;
+                break;
+            }
+        }
+
+        if (!$userFound) {
+            throw new \Exception(Piwik::translate('Cannot copy dashboard to user %s, user not found or user does not have access to same site.', $copyToUser));
```

Comment:
Guess this should be moved to a translation key, or the `Piwik::translate` can be removed

Reply:
didn't create translation key similar to the end of the API method where it is also not translated as when triggered through the UI this cannot happen... will remove the piwik translate